contracts/modules/capital/Pool.sol

```diff
@@ -102,13 +104,13 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
 
   /* ========== ASSET RELATED VIEW FUNCTIONS ========== */
 
-  function getAssetValueInEth(address assetAddress) internal view returns (uint) {
+  function getAssetValueInEth(address assetAddress, uint assetAmountInSwapOperator) internal view returns (uint) {
 
     uint assetBalance;
 
     if (assetAddress.code.length != 0) {
       try IERC20(assetAddress).balanceOf(address(this)) returns (uint balance) {
-        assetBalance = balance;
+        assetBalance = balance + assetAmountInSwapOperator;
```

Comment:
We could move this a few lines above to where we initialize the `assetBalance` uint. This way if the balance call fails we don't ignore the balance in the swap operator (although arguably we could as well ignore it already).